Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -55,16 +65,133 @@ def get_response(self, url, headers, payload, method='POST'):
         :param payload: Contains the request body
         """
 
-        try:
+        for _ in range(MAX_RETRIES):
 
-            if method == 'POST':
-                response = requests.request(method, url, headers=headers, json=payload).json()
+            try:
+                if method == 'POST' or method == 'PUT':
+                    response = requests.request(method, url, headers=headers, json=payload)
+                else:
+                    response = requests.request(method, url, headers=headers, params=payload)
+
+                response.raise_for_status()
+
+                response_json = response.json()
+                return response_json['data']
+
+            except (Exception,) as e:
```

Comment:
I am raising any exception because there isn't any need to catch it. If any error occurs, I just want the code to try again, that's why I have simply caught the error and allowed the code to pass.